tests/by-util/test_date.rs

```diff
@@ -279,7 +279,17 @@ fn test_date_for_invalid_file() {
     result.no_stdout();
     assert_eq!(
         result.stderr_str().trim(),
-        "date: invalid_file: No such file or directory",
+        "date: 'invalid_file': No such file or directory",
```

Comment:
Have you checked GNU? Do they quote? If not then you can just remove the `quote` call above from the code I suggested.

Reply:
So I ran a couple of checks on GNU.
```sh
➜  ~ date -f ~/foo1
date: /home/suryateja/foo1: No such file or directory
➜  ~ date -f ~/foo 
date: /home/suryateja/foo: Permission denied
➜  ~ date -f ~/foo.txt
date: invalid date ‘it\’s disabled’
➜  ~ 
```

They don't seem to be adding the quotes however they are returning the full path for said invalid file.  

Do I still remove the quote and update the test?

Reply:
Is the full path not just the `~` expanded by your shell?  What happens if you don't include the `~/`?

Reply:
But yeah, if you could make it compatible that would be great!